hera_pspec/pspecdata.py

```diff
@@ -613,7 +604,7 @@ def p_hat(self, M, q):
         """
         return np.dot(M, q)
 
-    def pspec(self, bls, weights='none', taper='none', verbose=False):
+    def pspec(self, bls, input_data_weight='identity',norm='I', taper='none', verbose=False):
```

Comment:
`input_data_weight` is a bit wordy -- can we get away with just calling it `data_weight`?